tofhir-rxnorm/src/main/scala/io/tofhir/rxnorm/PullRxNormNdcMedDetails.scala

```diff
@@ -0,0 +1,143 @@
+package io.tofhir.rxnorm
+
+import java.io.{FileReader, FileWriter}
+import com.opencsv.{CSVReaderBuilder, CSVWriter}
+import io.onfhir.api.util.FHIRUtil
+import org.json4s.JsonAST.{JArray, JDouble, JObject, JString}
+import org.json4s._
+import io.onfhir.util.JsonFormatter._
+
+object PullRxNormNdcMedDetails extends App {
```

Comment:
What does this class do? Please add a detailed comment to describe. Please give input and output examples.